File: loadtest/submission/scripts/villain/c/long_output.c
#include <stdio.h>
int main() {
    for(int i=0; i<1000000; i++) {
        printf("Output line %d: ", i);
        for(int j=0; j<100; j++){ 
            printf("a"); 
        }
        printf("\\n");
    }
    return 0;
}

File: loadtest/submission/scripts/villain/java/long_output.java
public class Main {
    public static void main(String[] args) {
        for(int i=0; i<1000000; i++) {
            System.out.print("Output line " + i + ": ");
            for(int j=0; j<100; j++){ 
                System.out.print("a"); 
            }
            System.out.println();
        }
    }
}